package/src/ApiContext/index.tsx

```diff
@@ -240,6 +256,12 @@ const APIProvider: React.FC<APIProviderType> = (props) => {
       let rawResponseGateways: GatewaysResponse;
       let responseGateways: GatewaysResponse;
       try {
+        if (props.selectGatewayBy === "performance") {
```

Comment:
instead of using the string ""performance" , best to use an ENUM. or a const string value.